common/scripted_triggers/wc_clothing_triggers.txt

```diff
@@ -91,7 +91,7 @@ portrait_fp1_dwarven_clothing_spouse_trigger = {
 	portrait_dwarven_clothing_spouse_trigger = yes
 }
 portrait_dwarven_hairstyles_trigger = {
-	scope:culture = { has_clothing_gfx = dwarf_hairstyles_gfx }
+	# scope:culture = { has_clothing_gfx = dwarf_hairstyles_gfx }
```

Comment:
dwarf_hairstyles_gfx is currently not in use. I commented it in case we need it in the future.
![image](https://user-images.githubusercontent.com/77478662/211453460-5d028409-f6fa-46bf-ad51-f756d77fba20.png)

Reply:
Used too.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/213882168-211c1eaa-3977-4ad4-9d0b-f3f7be6befc3.png)

</details>

Reply:
Yes, but no cultures have it assigned (due to unfinished/missing models?). That's why there are errors.

Reply:
The same mistake. The dwarves should have it.